tests/foreman/cli/test_repository.py

```diff
@@ -201,6 +203,154 @@ def test_positive_create_with_auth_yum_repo(self):
                 self.assertEqual(new_repo['content-type'], u'yum')
 
     @tier1
+    def test_positive_create_with_download_policy(self):
+        """Create YUM repositories with available download policies
+
+        @id: ffb386e6-c360-4d4b-a324-ccc21768b4f8
+
+        @Assert: YUM repository with a download policy is created
+        """
+        for policy in DOWNLOAD_POLICIES:
+            new_repo = self._make_repository({
+                u'content-type': u'yum',
+                u'download-policy': policy
+            })
+            self.assertEqual(new_repo['download-policy'], policy)
+
+    @tier1
+    def test_positive_create_with_default_download_policy(self):
+        """Verify if the defaualt download policy is assigned
+        when creating a YUM repo without `--download-policy`
+
+        @id: 9a3c4d95-d6ca-4377-9873-2c552b7d6ce7
+
+        @Assert: YUM repository with a default download policy
+        """
+        new_repo = self._make_repository({u'content-type': u'yum'})
+        self.assertEqual(new_repo['download-policy'], 'immediate')
+        # get the setting instead?
```

Comment:
I guess the comment is about having the `default policy` as a setting variable somewhere because if is going to change to `on_demand` in future no?